docs/compiling-coreboot.md

```diff
@@ -30,10 +30,10 @@ Building and flashing your own firmware has the potential to brick your device.
       * For example, `./build-uefi.sh leona`
     * Roms will be stored in `~/dev/firmware`
 6. **Download flashrom, then give it execute permission.**
-    * `cd; curl -LO https://elly.rocks/tmp/coreboot-development/flashrom-alderlake; chmod +x flashrom-alderlake`
+    * `cd; curl -LO https://cdn.discordapp.com/attachments/232609321274310656/1168575883980836894/flashrom; chmod +x flashrom`
```

Comment:
Maybe link to tree123.org or ethanthesleepy.one instead of discord

Reply:
Yeah linking to discord is stupid, I have that file here https://tree123.org/chrultrabook/utils/flashrom-libpci38.